src/kuzya.cpp

```diff
@@ -1101,8 +1243,12 @@ bool Kuzya::slotSaveChangesNotifier(void)
         {
             return false;	// false - cancelbtn pressed
         }
-        if (msgBox->clickedButton() ==yesBtn)
+        if (msgBox->clickedButton() == yesBtn)
         {
+            if(!myFile->isSaved())
+            {
+                fileName = QString::null;
```

Comment:
Use QString::clear().

Reply:
This is fileName = QString :: clear (); causes compilation error. Used fileName.clear ();